src/csharp/Grpc.Core.Api/ChannelCredentials.cs

```diff
@@ -51,6 +52,23 @@ public static ChannelCredentials Insecure
             }
         }
 
+        /// <summary>
+        /// Returns instance of credentials that provides SSL security.
+        /// <para>
+        /// When used with Grpc.Core, these credentials will load from a
+        /// disk file pointed to by the GRPC_DEFAULT_SSL_ROOTS_FILE_PATH environment variable.
+        /// If that fails, gets the roots certificates from a well known place on disk.
+        /// Use <see cref="SslCredentials"/> to customize the credentials.
+        /// </para>
+        /// </summary>
+        public static ChannelCredentials Secure
```

Comment:
I was also thinking of names such as `SecureSsl` or `SecureTls`, but than rejected `SecureTls` for naming consistency with pre-existing APIs and other gRPC implementations (even though SslCredentials actually create TLS connections).
I'm undecided between `SecureSsl` and `Secure`.  Would `SecureSsl` make sense in the context of grpc-dotnet? If so, that's probably a better name since with gRPC, "secure" doesn't always mean SSL/TLS (there are other transports that support other auth mechanisms than SSL/TLS).

Reply:
`SecureSsl` is fine with me. Updated.